.gitignore

```
@@ -67,6 +67,7 @@ typings/
 
 # dotenv environment variables file
 .env
+.env*.local
```

```suggestion
.env*
!.env.defaults
```
Ignore all `.env` files except for `.env.defaults` instead of ignore `.env` and all `.env*.local` files.

---

Irrelevant suggestion, I get it now, only the local files are supposed to hold values that should not be version controlled. As per "[files under version control](https://www.npmjs.com/package/dotenv-flow#files-under-version-control)".

---

Would it be fair to actually remove .env from `.gitignore`?

Per: ".env – for default (fallback) values, tracked by VCS" - See: [Files under version control](https://www.npmjs.com/package/dotenv-flow#files-under-version-control).

---

Done, and renamed `.env.defaults` to `.env` as requested by @ajacksified in Slack: b2e8ffea66e112ac337b469bd9312e9f44055376.